hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieLockConfig.java

```diff
@@ -189,6 +189,19 @@ public class HoodieLockConfig extends HoodieConfig {
       .withDocumentation("Lock provider class name, this should be subclass of "
           + "org.apache.hudi.client.transaction.ConflictResolutionStrategy");
 
+  public static final ConfigProperty<Boolean> RETRY_ON_CONFLICT_FAILURES = ConfigProperty
+      .key(LOCK_PREFIX + "retry.on.conflict.failures")
+      .defaultValue(false)
+      .sinceVersion("0.13.0")
+      .withDocumentation("Whenever there is a conflict, one of the commits will get aborted. By enabling this conflict, hudi will retry the batch again.");
+
+  public static final ConfigProperty<Integer> NUM_RETRIES_ON_CONFLICT_FAILURES = ConfigProperty
```

Comment:
There is already a retry config for acquiring lock. Why can't we reuse that? Is the intention of this config to retry for any kind of failure and not just lock acquisition? If so, would it be better to add it in write configs instead of lock configs?